### PR TITLE
Fixes identity provider section

### PR DESCRIPTION
**Pull request by ikhomyn**
## Motivation
Fixes for #1756 , #1754 & #1753 

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: src/user/UserIdentityProviderLinks.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ export const UserIdentityProviderLinks = () => {
     )?.groupName!;
     return (
       <Label color={groupName === "Social" ? "blue" : "orange"}>
-        {groupName === "Social" ? "Social" : "Custom"}
+        {groupName === "Social" ? "Social login" : "Custom"}
```
Comment (jonkoops) [inline src/user/UserIdentityProviderLinks.tsx:128]: Perhaps we should be localizing these strings.

File: src/user/UserIdentityProviderLinks.tsx [hunk 2]
```diff
@@ -136,7 +136,11 @@ export const UserIdentityProviderLinks = () => {
     )?.groupName!;
     return (
       <Label color={groupName === "User-defined" ? "orange" : "blue"}>
-        {groupName === "User-defined" ? "Custom" : groupName!}
+        {groupName === "User-defined"
```
Comment (jonkoops) [inline src/user/UserIdentityProviderLinks.tsx:139]: Same here.

File: src/user/user-section.css [hunk 3]
```diff
@@ -121,4 +122,19 @@ article.pf-c-card.pf-m-flat.kc-available-idps > div > div > h1 {
...
+  padding-bottom: 0px;
+}
+
+.pf-c-card__body.kc-form-panel__body{
+  padding-bottom: 0px;
+}
+
+.pf-c-content.kc-no-providers-text{
+  margin-top: 40px;
+  margin-right: 16px;
```
Comment (jonkoops) [inline src/user/user-section.css:137]: Perhaps we can use the PatternFly spacing variables here?

File: src/user/user-section.css [hunk 4]
```diff
@@ -121,4 +122,19 @@ article.pf-c-card.pf-m-flat.kc-available-idps > div > div > h1 {
 
 .keycloak__user-credentials__reset-form {
   --pf-c-form--m-horizontal__group-label--md--GridColumnWidth: 13rem;
-}
\ No newline at end of file
+}
+
+.pf-c-card__header.kc-form-panel__header{
+  padding-bottom: 0px;
```
Comment (jonkoops) [inline src/user/user-section.css:128]: You can omit the unit if the value is `0`.
```suggestion
  padding-bottom: 0;
```

File: src/user/user-section.css [hunk 5]
```diff
@@ -121,4 +122,19 @@ article.pf-c-card.pf-m-flat.kc-available-idps > div > div > h1 {
...
+}
+
+.pf-c-card__body.kc-form-panel__body{
+  padding-bottom: 0px;
```
Comment (jonkoops) [inline src/user/user-section.css:132]: Same here.
```suggestion
  padding-bottom: 0;
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/user/UserIdentityProviderLinks.tsx (hunk 1 above)
Comment (ikhomyn) [reply src/user/UserIdentityProviderLinks.tsx]: I have added localization for those. PTAL. Thank you @jonkoops

File: src/user/user-section.css (hunk 3 above)
Comment (ikhomyn) [reply src/user/user-section.css]: Changed as well.

File: src/user/user-section.css [hunk 6]
```diff
@@ -121,4 +122,19 @@ article.pf-c-card.pf-m-flat.kc-available-idps > div > div > h1 {
...
+.pf-c-card__header.kc-form-panel__header{
+  padding-bottom: 0;
+}
+
+.pf-c-card__body.kc-form-panel__body{
+  padding-bottom: 0;
+}
+
+.pf-c-content.kc-no-providers-text{
+  margin-top: 40px;
```
Comment (jonkoops) [inline src/user/user-section.css:136]: Could we also use PatternFly variables here?

**Review by jonkoops — APPROVED** [review]
One small comment, but LGTM

File: src/user/user-section.css (hunk 6 above)
Comment (ikhomyn) [reply src/user/user-section.css:136]: I was unable to find any variable having 40px. Should I use this one --pf-global--spacer--2xl? which is 48px ?

Comment (ikhomyn) [reply src/user/user-section.css]: @jonkoops I found a way. Now should be ok

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@ikhomyn can #1756, #1754 and #1753 be closed?

**Comment by jonkoops** [comment]
No worries, I closed the issues thanks!

**Pull request merged**